workloads/vars/http.yml

```diff
@@ -28,6 +28,10 @@ job_completion_poll_attempts: "{{ lookup('env', 'JOB_COMPLETION_POLL_ATTEMPTS')|
 
 # HTTP workload specific parameters
 http_env_vars: [
+  "ES_SERVER",
```

Comment:
The elastic search vars needs to be added to the docs.

Reply:
Done